ssl/quic/quic_impl.c

```diff
@@ -1696,12 +1700,16 @@ static SSL *quic_conn_stream_new(QCTX *ctx, uint64_t flags, int need_lock)
     }
 
     qs = ossl_quic_channel_new_stream_local(qc->ch, is_uni);
-    if (qs == NULL)
+    if (qs == NULL) {
+        ERR_raise(ERR_LIB_SSL, ERR_R_INTERNAL_ERROR);
```

Comment:
These should 100% be using `QUIC_RAISE_NON_NORMAL_ERROR`. This macro should be used for anything dispatched via an API function (i.e., via `ssl_lib.c`) or where one of the `expect_quic` family of functions has been called on the call path. I'd be surprised if there are valid cases for direct calls to ERR_raise in `quic_impl.c` in particular. It would certainly be unusual.